extensions/amp-subscriptions/0.1/crypto-handler.js

```diff
@@ -92,11 +92,36 @@ export class CryptoHandler {
    * @return {Promise<string>}
    */
   decryptDocumentContent_(
-    // eslint-disable-next-line no-unused-vars
     encryptedContent, documentKey) {
-    // Don't really return this. Placeholder for the real thing.
-    // const placeholder = encryptedContent.trim() + documentKey;
-    const placeholder = '<h2><i> abc </i></h2>';
-    return Promise.resolve(placeholder);
+
+console.log('encryptedContent', encryptedContent);
+console.log('documentKey', documentKey);
+
+    const decryptedContent = crypto.subtle.decrypt(
+        {
+          name: 'AES-CTR',
+          counter: new Uint8Array(16), // iv: all zeros.
+          length: 128, // block size (16): 1-128
+        },
+        this.base64ToBytes(documentKey),
+        encryptedContent.trim(),
```

Comment:
We can't just decrypt the text. First we need to extract bytes from it. Here's how we should approach it:

1. We need to trim and remove all whitespaces in between (e.g. line breaks). This way we get the complete encoded and encrypted string.
2. Un-base64 the text content. This way we get the actual crypted bytes.
3. Call `decrypt`.